src/index.ts

```diff
@@ -135,14 +141,22 @@ async function init(): Promise<void> {
 
   const apiServer = await startApiServer({ datastore: db, chainId: getConfiguredChainID() });
   logger.info(`API server listening on: http://${apiServer.address}`);
-  registerShutdownHandler(async () => {
-    await apiServer.terminate();
+  registerShutdownConfig({
+    name: 'API Server',
```

Comment:
(for notes) this one and the other handlers that I didn't comment on are _not_ critical and can be force closed